Test.Utility/TypesThatNeedToBeImplementedInAssemblyDataAttribute.cs

```diff
@@ -47,8 +49,20 @@ public TypesThatNeedToBeImplementedInAssemblyDataAttribute(string assemblyName)
         }
 
         public override IEnumerable<object[]> GetData(MethodInfo testMethod)
-            => TypesThatNeedToBeImplementedInAssemblyRetriever
+        {
+            var excludedTypes = CollectExcludedTypes(testMethod);
+            return TypesThatNeedToBeImplementedInAssemblyRetriever
                 .GetTypesThatNeedToBeImplementedInAssembly(_assemblyName)
+                .Where(type => !excludedTypes.Contains(type))
                 .Select(type => new object[] { type });
+        }
+
+        private static ISet<Type> CollectExcludedTypes(MethodInfo testMethod)
+        {
+            var types = testMethod
+                .GetCustomAttributes<ExcludedTypesAttribute>()
+                .SelectMany(attr => attr.Types);
+            return new HashSet<Type>(types);
```

Comment:
You can turn this into a single expression by usind [.ToHashSet](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.tohashset?view=netframework-4.8)

Reply:
That’s not supporter on dotnet standard 2.0